core/src/main/java/com/novoda/noplayer/mediaplayer/TrackInfosFactory.java

```diff
@@ -0,0 +1,20 @@
+package com.novoda.noplayer.mediaplayer;
+
+import android.media.MediaPlayer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class TrackInfosFactory {
+
+    NoPlayerTrackInfos createFrom(MediaPlayer mediaPlayer) {
```

Comment:
Transformation from `MediaPlayer.TrackInfo` to our `NoPlayerTrackInfo` wrapped in a first class collection.